scopes/scope/importer/import.cmd.ts

```diff
@@ -53,37 +53,45 @@ export class ImportCmd implements Command {
     [
       'o',
       'objects',
-      'import components objects to the local scope without checkout (without writing them to the file system). This is a default behavior for import with no id argument',
+      'import components objects to the local scope without checkout (without writing them to the file system). This is the default behavior for import with no id argument',
     ],
-    ['d', 'display-dependencies', 'display the imported dependencies'],
+    ['d', 'display-dependencies', 'display the imported dependencies'], // @david what are the imported dependencies?
     ['O', 'override', 'override local changes'],
     ['v', 'verbose', 'show verbose output for inspection'],
     ['j', 'json', 'return the output as JSON'],
     // ['', 'conf', 'write the configuration file (component.json) of the component'], // not working. need to fix once ComponentWriter is moved to Harmony
-    ['x', 'skip-dependency-installation', 'do not install packages of the imported components'],
+    ['x', 'skip-dependency-installation', 'do not auto-install dependencies of the imported components'],
     [
       'm',
       'merge [strategy]',
       'merge local changes with the imported version. strategy should be "theirs", "ours" or "manual"',
     ],
-    ['', 'dependencies', 'import all dependencies and write them to the workspace'],
+    [
+      '',
+      'dependencies',
+      'import all dependencies (bit components only) of imported components and write them to the workspace',
+    ],
     [
       '',
       'dependents',
-      'import components found while traversing from the given ids upwards to the workspace components',
+      'import components found while traversing from the imported components upwards to the workspace components',
     ],
     [
       '',
       'save-in-lane',
-      'when checked out to a lane and the component is not on the remote-lane, save it in the lane (default to save on main)',
+      'when checked out to a lane and the component is not on the remote-lane, save it in the lane (defaults to save on main)',
     ],
     [
       '',
       'all-history',
       'relevant for fetching all components objects. avoid optimizations, fetch all history versions, always',
     ],
-    ['', 'fetch-deps', 'fetch dependencies objects'],
-    ['', 'track-only', 'do not write any file, just create .bitmap entries of the imported components'],
+    ['', 'fetch-deps', 'fetch dependencies (bit components) objects to the local scope, but dont add to the workspace'], // @david is this edit correct? If so, what does this do?
```

Comment:
`fetch` only downloads the objects to the local `.bit` directory. behind the scenes `import` runs this, and checkout to the worksapce.

think about the diff between `git fetch` and `git pull`.

Reply:
bit import only checks out to the component when run on a single component, but this flag is available for all runs of bit import.
So basically this flag is default behaviour for `bit import` without specifying a component?

Reply:
i don't know. need to see with @davidfirst

Reply:
In the past we used to fetch dependencies always.
Now, with the recent optimizations, the dependencies graph is saved into each one of the versions, so we don't fetch dependencies by default, unless you enter this flag.
This is true for all kind of imports (patterns/ids/no-param)

Reply:
OK thanks for the description
Follow up - does it add the deps to the workspace, or just fetch their objects? Where are these objects then used (i.e. why would i do this)?

Reply:
It only fetches the objects, it doesn't write them to the workspace.
It can be useful when getting errors about missing components/dependencies for example.

Reply:
ok, i've tried to add that to the description - @davidfirst please confirm it makes sense and then can resolve this conversation

Reply:
Yup. looks good.